libraries/ui-shared/ScannerOverlay/components/ScannerBar/components/FlashlightButton/index.jsx

```diff
@@ -24,7 +24,7 @@ const FlashlightButton = ({
       <ToggleIcon
         on={flashlightState}
         onIcon={<FlashOnIcon className={styles.icon} />}
```

Comment:
Can we please rename the "on" icon to `FlashEnabledIcon` for consistency, also considering the public API?

Reply:
Sure